wiki/Mapping_Techniques/Inspiration_Gallery_Corner/en.md

```diff
@@ -0,0 +1,176 @@
+# Inspiration Gallery Corner
+
+> blame naraku
+
+Here is a gallery of miscellaneous and interesting sliders.
+
+## narakucrimson's Sliders of Wonderment
+
+![](img/narakucrimson-1.png)
+
+![](img/narakucrimson-2.png)
+
+![](img/narakucrimson-3.png)
+
+## "U" Slider
+
+![](img/u.png)
+
+## Semi-Elbow
+
+![](img/semi-1.png)
+
+![](img/semi-2.png)
+
+## Awesome Wiggles
+
+![](img/awesome-1.png)
+
+![](img/awesome-2.png)
+
+## Interesting Heart-Shaped Slider
+
+![](img/interesting.png)
+
+## Slider Blankets
+
+![](img/slider-1.png)
+
+![](img/slider-2.png)
+
+![](img/slider-3.png)
```

Comment:
L41: Place a text inside ``[]`` to indicate which was supposed to be there if the image fails to load.